campaigns/mofo-eoy-2015/snippet.html

```diff
@@ -266,6 +258,15 @@
                 var btnSelected;
                 var amount;
                 var presets = '{{donationAmountFirst}},{{donationAmountSecond}},{{donationAmountThird}},{{donationAmountFourth}}';
+                var numberFormat = new Intl.NumberFormat('{{locale}}', {
```

Comment:
`Intl.NumberFormat`, nice! this works for Fx 29+ though, we need to verify that this is OK with the team. I'll comment in the bug.